pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -447,6 +470,82 @@ func possibleIndexesForID(containerID string, mapping map[int]map[string]struct{
 	return possibles
 }
 
+func getContainerRootInUVM(id string) string {
+	switch runtime.GOOS {
+	case "windows":
+		return filepath.Join(guestpath.WCOWRootPrefixInUVM, id)
+	default:
+		return filepath.Join(guestpath.LCOWRootPrefixInUVM, id)
+	}
+}
```

Comment:
Kinda confused, is this run on the host or guest? Trying to think how this works if you ran on the host as we'd always get a wcow prefix

Reply:
removed this and just used the const directly. also updated docs to say that enforcement is expecting a linux system.